fuel-vm/src/tests/wideint.rs

```diff
@@ -0,0 +1,654 @@
+use ethnum::U256;
```

Comment:
looks like we're missing tests for WDMD?

Reply:
Added those. I feel like adding copy-pasted tests for macro-generated code isn't too useful, but still nice to have for future proofing.